app/controllers/merchant/items_controller.rb

```diff
@@ -1,4 +1,25 @@
 class Merchant::ItemsController < Merchant::BaseController
   def index
+    @merchant = Merchant.find(current_user.merchant_id)
```

Comment:
This is beautiful!

Reply:
I love the use of the current_user method and then accessing that merchant id from it!